src/Connector.js

```diff
@@ -364,24 +364,12 @@ class Connector {
 
     const sensors = await Promise.all(schemaList.map(async (schema) => {
       await this.client.subscribe(`/${id}/${schema.sensorId}/cmd`);
-
       return mapSensorToFiware(id, schema);
     }));
 
-    try {
-      await request.post({
-        url, headers, body: { devices: sensors }, json: true,
-      });
-    } catch (err) {
-      /*
-      The 409 error means the request couldn't be completed due to a conflict in
-      respect to the target resource - contextually, a (deviceId, serviceId) pair
-      already exists. However, this occurs when attempting to make changes in a
-      resource which the base connector is already acting upon due to a previous
-      request (e.g. updateSchema). As soon as the request is processed, the error
-      will be gone, without consequences; therefore, it shouldn't cause an interuption.
-      */
-    }
+    await request.post({
```

Comment:
About commit message 80a20e51ba1699f9db091fd617b39a542b764981:

I think that you've created this commit message before edit the commit message that you referenced. (When you use the `commit --amend` the hash changes)

The correct hash is:
... was corrected in the commit 549bd2501ba46a8c87509d06ee0b9c1c99843912

Reply:
Wow, I totally let that pass by unnoticed 👀 
Sorry for that! Updated commit message, as requested 😉